### DISCUSSION TITLE
Connection Error 104

### DISCUSSION THREAD

**Discussion by PumpkinsLemon** [fastapi/fastapi#11174] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
gunicorn -w $1 -b 0.0.0.0:${PORT} --max-requests 1000  --max-requests-jitter 100 --timeout 300 -k uvicorn.workers.UvicornH11Worker service_v2.main:app --error-logfile log/error.log --log-level warning
```

### Description

So i have a project that was made back using FastAPI and I deployed it on containers using gunicorn and unicorn.
<img width="1821" alt="image" src="https://github.com/tiangolo/fastapi/assets/47110984/7d7c2c49-3f8c-4aff-b0b2-f445d68a31ef">now, connection Error 104 is causing me a lot of distress
`recv() failed (104: Connection reset by peer) while reading response header from upstream`
What measures should I take to solve this problem

### Operating System

Linux

### FastAPI Version

0.83.0

### Pydantic Version

1.9.2

### Python Version

3.6.8

**Comment by estebanx64** [comment]
First of all can you provide the gunicorn logs, they should be in some log/error.log var/logs.
Second one, are you behind a NGINX/Traefik Proxy? it may be a configuration between the proxy and gunicorn too

**Reply by PumpkinsLemon** [reply]
Yes, this error occurred between the nginx agent and my service. Second Gunicorn logs have no content, and these requests were not actually sent to my service provider.

**Reply by estebanx64** [reply]
can you leave here your nginx configuration file?

**Reply by PumpkinsLemon** [reply]
My nginx configuration adopts the default configuration. It is worth noting that there is a significant difference between the timeout time (60s) and the timeout time of my service (300s)

**Comment by YuriiMotov** [comment]
Let's close this for now to keep the discussions clean and organized.
If you’ve found a solution, please share it here.
If you still need help, feel free to let us know.
